src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/PniService.kt

```diff
@@ -68,8 +68,9 @@ class PniService(
     val assessmentId = assessmentIdDate.first
 
     // section 6
-    var completedAssessmentWithSara = oasysService.getAssessmentWithCompletedSara(prisonNumber) ?: assessmentId
-    val relationships = oasysService.getRelationships(completedAssessmentWithSara)
+    var completedAssessmentIdWithSara = oasysService.getAssessmentWithCompletedSara(prisonNumber)
```

Comment:
Get assessments is already called as part of `fun getAssessmentIdDate(prisonNumber: String): Pair<Long, LocalDateTime?>? {`

Is it possible to reuse it, rather than making the assessment call again?